be/src/olap/rowset/alpha_rowset.cpp

```diff
@@ -75,6 +90,10 @@ RowsetMetaSharedPtr AlphaRowset::rowset_meta() const {
 }
 
 void AlphaRowset::set_version(Version version) {
+    if (_removed) {
```

Comment:
I think Rowset is already removed in memory structure synchronously if _removed is true.
So this judgement will be redundant.